fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -549,12 +551,21 @@ public String getRemoteIp() {
             row.add(getMysqlChannel().getRemoteHostPortString());
             row.add(clusterName);
             row.add(ClusterNamespace.getNameFromFullName(currentDb));
+                        row.add(queryDetail.getQueryId());
             row.add(command.toString());
+            row.add(fromLongToDate(startTime));
             row.add("" + (nowMs - startTime) / 1000);
             row.add("");
-            row.add("");
+            String stmt = executor == null ? "" : executor.getOriginStmtInString();
```

Comment:
We can already view the currently running query through the function of ```SHOW PROC "/current_query_stmts"```. 
What is the main purpose of adding here?

Reply:
we can view current sql already running time,  can use the query_id to find log , for locate the perform problem, like postgresql: select * from pg_stat_activity